src/RightsService.Data/UserRepository.cs

```diff
@@ -94,5 +94,13 @@ public List<DbUser> Get(List<Guid> userId)
         {
             return _provider.Users.Where(u => userId.Contains(u.UserId)).Include(u => u.Role).ToList();
         }
+
+        public void Remove(Guid userId)
+        {
+            DbUser user = _provider.Users.FirstOrDefault(u => u.UserId == userId);
```

Comment:
What happened if we dont find user with that guid? mb do check?

Reply:
you are right, fixed